### PR TITLE
Add code coverage via coveralls

### PR DESCRIPTION
**Pull request by caniszczyk**
https://coveralls.io/r/twitter/twitter-cldr-rb

Signed-off-by: Chris Aniszczyk <zx@twitter.com>

### CODE REVIEW

**Comment by camertron** [comment]
Looks almost identical to #144, was that intentional?

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -6,6 +6,8 @@
 require 'rspec'
 require 'rspec/autorun' # somehow makes rcov work with rspec
 require 'twitter_cldr'
+require 'coveralls'
+Coveralls.wear!
```
Comment (camertron) [inline spec/spec_helper.rb]: According to the docs, `#wear!` causes results to be sent to Coveralls whenever the project is built on Travis. I did a cursory read of the source code and it didn't look like that was true... I must have missed something.

Also, twitter-cldr-rb uses Simplecov already. We might consider hooking up Coveralls' Simplecov integration.

Comment (caniszczyk) [inline spec/spec_helper.rb]: Feel free to do that as you're more familiar with Ruby than I am :)

Comment (camertron) [inline spec/spec_helper.rb]: Alright, I'll see what I can do :)

Comment (KL-7) [inline spec/spec_helper.rb]: I'm not sure how it works, but coveralls already has [a few reports](https://coveralls.io/r/twitter/twitter-cldr-rb) for this repo.

**Comment by caniszczyk** [comment]
Ya, those were me pushing things locally

File: spec/spec_helper.rb (hunk 1 above)
Comment (caniszczyk) [inline spec/spec_helper.rb]: Ya, those were me pushing things locally

File: README.md [hunk 2]
```diff
@@ -1,6 +1,4 @@
-
-
-## twitter-cldr-rb [![Build Status](https://secure.travis-ci.org/twitter/twitter-cldr-rb.png?branch=master)](http://travis-ci.org/twitter/twitter-cldr-rb) [![Code Climate](https://codeclimate.com/github/twitter/twitter-cldr-rb.png)](https://codeclimate.com/github/twitter/twitter-cldr-rb)
+## twitter-cldr-rb [![Build Status](https://secure.travis-ci.org/twitter/twitter-cldr-rb.png?branch=master)](http://travis-ci.org/twitter/twitter-cldr-rb) [![Code Climate](https://codeclimate.com/github/twitter/twitter-cldr-rb.png)](https://codeclimate.com/github/twitter/twitter-cldr-rb) [![Coverage Status](https://coveralls.io/repos/twitter/twitter-cldr-rb/badge.png)](https://coveralls.io/r/twitter/twitter-cldr-rb)
```
Comment (KL-7) [inline README.md]: Our readme is generated from `README.md.erb`, so all the changes should go in there. Then you run `bundle exec rake update:readme` and it updates `README.md`. Could you do that? If you don't have environment set up, I can help with that.

Comment (KL-7) [inline README.md]: I mean helping with updating the readme, not with setting up the environment.

**Comment by camertron** [comment]
@KL-7 do you have time to help @caniszczyk with this?

**Comment by KL-7** [comment]
Done :)

**Comment by caniszczyk** [comment]
DANKE!

**Comment by KL-7** [comment]
Alright, it's passing CI, good to merge? 

Also, I don't get what this coveralls bot is trying to say. Why is it `changes Unknown` and what's up with bold text and asterisks? If it's bugged, maybe we can disable it in the settings on coveralls.io?

**Comment by caniszczyk** [comment]
Not sure what's up with the "Changes Unknown"

Next step, lets speed up the build :)

**Pull request merged**